config/initializers/rack_attack.rb

```diff
@@ -69,17 +69,17 @@ def self.max_requests_per_second
     #   [503, {}, ['Server Error']] # status  # headers  # body
     # end
 
-    self.throttled_responder = lambda do |env|
+    self.throttled_responder = lambda do |request|
       now = Time.zone.now
-      match_data = env['rack.attack.match_data']
+      match_data = request.env['rack.attack.match_data']
 
       headers = {
         'X-RateLimit-Limit' => match_data[:limit].to_s,
         'X-RateLimit-Remaining' => '0',
         'X-RateLimit-Reset' => (now + (match_data[:period] - now.to_i % match_data[:period])).to_s
       }
 
-      [429, headers, ["Throttled\n"]]
+      [503, headers, ["Throttled\n"]]
```

Comment:
¿A considerar tal vez en el futuro?
El envío de los headers y la respuesta `"Throttled"` está documentada en el siguiente apartado del README de la gema: https://github.com/rack/rack-attack#ratelimit-headers-for-well-behaved-clients
Sería para clientes bien intencionados, y allí se documenta devolver un 429. 

La sugerencia del 503 entiendo que era para clientes mal intencionados, para hacerle creer que tiró abajo la API sería, y no tendría sentido devolverle ningun Header de RateLimit.
![image](https://github.com/widergy/rails-api-bootstrap/assets/53331055/fb34eb6c-7ea5-4a62-a88b-105322d88fdc)